test/spec/4_Playlists/4.3_Playlist-Tags/4.3.4_Master-Playlist-Tags/4.3.4.2_EXT-X-STREAM-INF.spec.js

```diff
@@ -143,6 +143,20 @@ test('#EXT-X-STREAM-INF_07-02', t => {
   `);
 });
 
+test('#EXT-X-STREAM-INF_07-03', t => {
```

Comment:
I thought what you really wanted to test was something this:
```
  const sourceText = `
  #EXTM3U
  #EXT-X-STREAM-INF:BANDWIDTH=1280000
  /video/main.m3u8
  `;
  const expectedText = `
  #EXTM3U
  #EXT-X-STREAM-INF:BANDWIDTH=1280000,CLOSED-CAPTIONS=NONE
  /video/main.m3u8
  `;
  let obj = HLS.parse(sourceText);
  let text = HLS.stringify(obj);
  t.is(text, utils.stripCommentsAndEmptyLines(sourceText));
  HLS.setOptions({allowClosedCaptionsNone: true});
  obj = HLS.parse(sourceText);
  text = HLS.stringify(obj);
  t.is(text, utils.stripCommentsAndEmptyLines(expectedText));
  HLS.setOptions({allowClosedCaptionsNone: false});
```

Reply:
@kuu I can include a test for when the setting is false.

Reply:
@hsume2 Thanks, please. I also think it's important to check if ",CLOSED-CAPTIONS=NONE" is added or not.

Reply:
Sorry, I think I misunderstood.
Will merge your final change.

Reply:
@kuu thank you for merging.  I can still add a test for when `allowClosedCaptionsNone: false` if you want?

Reply:
@hsume2 No need to add it, though I'm open to any improvements:) I misunderstood that your test would pass regardless of the change in stringify.js but it actually didn't.